src/librustc_borrowck/diagnostics.rs

```diff
@@ -314,6 +314,59 @@ let c = &i; // still ok!
 ```
 "##,
 
+E0504: r##"
+This error occurs when an attempt is made to move a borrowed variable into a
+closure. For example:
+
+```compile_fail
```

Comment:
I wondered about that-- I was trying to give an example that showed a common use of moved closures, but since it sounds like you think it would be clearer without I'll remove it.